src/arbitrum/utils/YieldOptimizersRegistry.sol

```diff
@@ -0,0 +1,273 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+// ███████╗░█████╗░██████╗░████████╗██████╗░███████╗░██████╗░██████╗
+// ██╔════╝██╔══██╗██╔══██╗╚══██╔══╝██╔══██╗██╔════╝██╔════╝██╔════╝
+// █████╗░░██║░░██║██████╔╝░░░██║░░░██████╔╝█████╗░░╚█████╗░╚█████╗░
+// ██╔══╝░░██║░░██║██╔══██╗░░░██║░░░██╔══██╗██╔══╝░░░╚═══██╗░╚═══██╗
+// ██║░░░░░╚█████╔╝██║░░██║░░░██║░░░██║░░██║███████╗██████╔╝██████╔╝
+// ╚═╝░░░░░░╚════╝░╚═╝░░╚═╝░░░╚═╝░░░╚═╝░░╚═╝╚══════╝╚═════╝░╚═════╝░
+// ███████╗██╗███╗░░██╗░█████╗░███╗░░██╗░█████╗░███████╗
+// ██╔════╝██║████╗░██║██╔══██╗████╗░██║██╔══██╗██╔════╝
+// █████╗░░██║██╔██╗██║███████║██╔██╗██║██║░░╚═╝█████╗░░
+// ██╔══╝░░██║██║╚████║██╔══██║██║╚████║██║░░██╗██╔══╝░░
+// ██║░░░░░██║██║░╚███║██║░░██║██║░╚███║╚█████╔╝███████╗
+// ╚═╝░░░░░╚═╝╚═╝░░╚══╝╚═╝░░╚═╝╚═╝░░╚══╝░╚════╝░╚══════╝
+
+//  _____         _                   _____     _   _ _____         _     _           
+// |   __|___ ___| |_ ___ ___ ___ ___|  _  |___| |_|_| __  |___ ___|_|___| |_ ___ _ _ 
+// |   __| . |  _|  _|  _| -_|_ -|_ -|     |  _| . | |    -| -_| . | |_ -|  _|  _| | |
+// |__|  |___|_| |_| |_| |___|___|___|__|__|_| |___|_|__|__|___|_  |_|___|_| |_| |_  |
+//                                                             |___|             |___|
+
+// Github - https://github.com/FortressFinance
+
+contract YieldOptimizersRegistry {
+
+    enum AMMType {
+        Curve,
+        Balancer
+    }
+
+    struct TargetAssets {
+        fortETH,
+        fortUSD,
+        fortCrypto1, 
+        fortCrypto2
```

Comment:
Strategically speaking, my opinion is we should Concentrate into 4ish assets at a maximum, those 4 assets should be `ETH` for added ETH exposure, `USD` for added USD exposure, and then assets which provide more exposure to the market as a whole, my personal favorites ATM are GLP and Curve's TriCrypto.

So `Crypto1` and `Crypto2` refers to the assets the provide exposure to the market as a whole - ATM it will be GLP and TriCrypto. Named them `Crypto1` and `Crypto2` just to leave us the option to change them later.

LMK what you think.

cc @xh3b4sd

Reply:
I guess there are many roads that lead to Rome. Naming is difficult in general and I do not have a really better suggestion for this. Using structs here works if you deal with the naming. My intuition would have been to use a list and cap it which would imply working with indices. That approach would then come with its own challenges.